pkg/sync/v5/default.go

```diff
@@ -108,5 +108,8 @@ func defaults(o unstructured.Unstructured) {
 	case "StorageClass.storage.k8s.io":
 		jsonpath.MustCompile("$.reclaimPolicy").DeleteIfMatch(o.Object, "Delete")
 		jsonpath.MustCompile("$.volumeBindingMode").DeleteIfMatch(o.Object, "Immediate")
+
+	case "ImageStream.image.openshift.io":
+		jsonpath.MustCompile("$.spec.tags.*.referencePolicy.type").DeleteIfMatch(o.Object, "Source")
```

Comment:
add something like:
```golang
jsonpath.MustCompile("$.spec.tags.*.referencePolicy.type").DeleteIfMatch(o.Object, "")
```

may also need something like the below, not sure:
```golang
jsonpath.MustCompile("$.spec.tags.*.referencePolicy").DeleteIfMatch(o.Object, map[string]interface{}{})
```

(default is mainly about removing excess baggage from our yaml files on disk)

Reply:
will also need to call default() from content_test.go